mdf.md

```diff
@@ -0,0 +1,91 @@
+#  MVDS Metadata Field
+
+> Version: 0.1.0 (Draft)
+> 
+> Authors: Oskar Thorén <oskar@status.im>, Dean Eigenmann <dean@status.im>
+
+##  Table of Contents
+
+1. [Abstract](#abstract)
+2. [Motivation](#motivation)
+3. [Format](#format)
+    1. [Fields](#fields)
+4. [Usage](#usage)
+    1. [Informational Fields](#informational-fields)
+    2. [Configurational Fields](#configurational-fields)
+
+## Abstract
+
+In this specification, we describe a method to construct both a linear and DAG (Directed Acyclic Graph) based message history that will aid the consistency of [MVDS](./mvds.md), which currently do not exist, casual consistency is added with the DAG and sequential consistency with the linear history. Additionally we explain how data sync can be used for more lightweight messages that do not require full synchronization.
```

Comment:
Typo throughout causal is misspelled casual.

I don't think this is accurate to say, clearly this is not enough to achieve causal consistency (if only was that easy :) )

1) We are passing messages that have unmet dependencies to the client (i.e missing parents)
2) If we bound the history of dependencies (as mentioned above), you will not have causal consistency (what happens if a missing message is outside the window?), but I understand we can reasonably bound this number.

So we need to have at least a combination of 1 & 2, basically you need to withhold messages until all dependencies are satisfied, and have an unbound way to specify dependencies, as a message can have an arbitrary number of parents (not ancestors) strictly speaking, up to number of members in a group (which is unbound).

I mentioned this many many times before, it seems there some fundamental misunderstanding of how causal consistency is achieved and some knowledge gaps.

Also I don't think is necessarily desirable given the consequences in all scenarios,
 certain not for a chat application, you are fine showing messages that have unmet dependencies but you want those dependencies to be fetched still.
So status chat would definitely not take advantage of this feature if causal consistency was enforced, only maybe for multi-part messages, but we don't have that use case.
Briar uses this in very limited scenarios (replies), and allows only a single parent for each message, which makes a huge assumption (a comment is only causally related to a line of their ancestry, which is self evidently not true in the real world).

This at best allows you to have eventual consistency ( or casual :) ) by travelling the dependency chain and eventually get to a consistent state, so if we want to merge as draft, sure, but we can't make claim that are clearly not supported.

Reply:
Agree, let's change language to be more about "enabling" and then be clear about requirements and implications

We can have an implementation/client recommendation section that is TBD, and mentions some caveats for when it makes sense or not.